src/otx/algorithms/classification/adapters/mmcls/models/heads/custom_vision_transformer_head.py

```diff
@@ -38,3 +38,7 @@ def post_process(self, pred):
         """Post processing."""
         pred = cast_bf16_to_fp32(pred)
         return super().post_process(pred)
+
+    def forward(self, x):
```

Comment:
It's just a question. Does error occur if this function doesn't exist?

Reply:
Yes, forward is required.

Reply:
https://mmpretrain.readthedocs.io/en/stable/api/generated/mmpretrain.models.heads.VisionTransformerClsHead.html#mmpretrain.models.heads.VisionTransformerClsHead.forward

Actually, VisionTrancformerHead should have a forward method
I'm not sure, that self.simple_test is the right choice for forward function

Reply:
You should check mmcls version adapted on OTX. 0.25.0 doesn't have any forward function.
https://github.com/open-mmlab/mmpretrain/blob/v0.25.0/mmcls/models/heads/vision_transformer_head.py